api/ApiHandler.py

```diff
@@ -1,21 +1,34 @@
-from flask_restful import Api, Resource, request
-
-class ApiHandler(Resource):
-  def get(self):
-    search_term = request.args.get("query")
-    ## TODO: return err if query is invalid twitter handle
-    ## TODO: use search_term to return proper array of words
-    return {
-      'resultStatus': 'SUCCESS',
-      'words': [
-                {
-                    "text": 'told',
-                    "value": 64
-                },
-                {
-                    "text": "hi", 
-                    "value": 1
-                }
-            ]
-      }
+from flask_restful import Resource
+import requests;
+import os;
 
+from flask import Flask, jsonify, json
+
+
+class ApiHandelr(Resource):
+  url = 'https://api.twitter.com/1.1/statuses/user_timeline.json'
+  Twitter_Barear_Token= os.environ.get('Twitter_Barear_Token')
+
+  header = {"Authorization":  f"Bearer {Twitter_Barear_Token}"}
+  def getTweets(self):
+      userName = requests.args.get("userName")
+      searchWord=  requests.args.get("word")
+      params = {'screen_name': userName};
+      response = requests.get(url, auth= header, params = params)
```

Comment:
I would create a separate file like "twitter.py" to hold the api integration. then you can call out a helper method from the api endpoint to make the call to the twitter api. I'm envisioning something like: 

at the top of the file: 
`import get_tweets from twitter.py`

within the get endpoint: 

tweets = get_tweets(username)

Then after you get the raw tweet data back, you'll need to parse that to the format the wordcloud expects, which we can do via a call to another helper, so the next line could be for example:

parsed_tweets = format_tweets(tweets) or something :) 

Note that snake case is more pythonic!